zipline/finance/controls.py

```diff
@@ -425,3 +425,48 @@ def validate(self,
         """
         if _account.leverage > self.max_leverage:
             self.fail()
+
+
+class MinLeverage(AccountControl):
+    """
+    AccountControl representing a limit on the minimum leverage allowed
+    by the algorithm after a threshold period of time.
+    """
+
+    def __init__(self, min_leverage, deadline):
+        """
+        max_leverage is the gross leverage in decimal form. For example,
+        2, limits an algorithm to trading at most double the account value.
+        """
+        super(MinLeverage, self).__init__(min_leverage=min_leverage,
+                                          deadline=deadline)
+        self.min_leverage = min_leverage
+        self.deadline = deadline
+
+        if min_leverage is None:
+            raise ValueError(
+                "Must supply min_leverage"
+            )
+
+        if min_leverage < 0:
+            raise ValueError(
+                "min_leverage must be positive"
+            )
+
+        if deadline is None:
+            raise ValueError(
+                "Must supply deadline"
+            )
+
+    def validate(self,
+                 _portfolio,
```

Comment:
The underscores here are a weird circa-2014 scott-ism: the contention elsewhere in this file is that the paramters start with an underscore if they're unused in the function, which was a convention I'd used in a previous codebase.  We don't do that anywhere else in zipline though.  I'd either follow the existing convention in this file (in which case `algo_datetime` and `account` shouldn't have underscores), or just make them all normal-looking parameters.